Core/Security/X509Chain.py

```diff
@@ -125,92 +125,92 @@ def setPKey( self, pkeyObj ):
     self.__loadedPKey = True
     return S_OK()
 
-  def loadProxyFromFile( self, chainLocation ):
+  def loadProxyFromFile(self, chainLocation):
     """
     Load a Proxy from a pem file
     Return : S_OK / S_ERROR
     """
     try:
-      fd = file( chainLocation )
+      fd = file(chainLocation)
       pemData = fd.read()
       fd.close()
     except Exception as e:
-      return S_ERROR( DErrno.EOF, "%s: %s" % ( chainLocation, repr( e ).replace( ',)', ')' ) ) )
-    return self.loadProxyFromString( pemData )
+      return S_ERROR(DErrno.EOF, "%s: %s" % (chainLocation, repr(e).replace(',)', ')')))
+    return self.loadProxyFromString(pemData)
 
-  def loadProxyFromString( self, pemData ):
+  def loadProxyFromString(self, pemData):
     """
     Load a Proxy from a pem buffer
     Return : S_OK / S_ERROR
     """
-    retVal = self.loadChainFromString( pemData )
-    if not retVal[ 'OK' ]:
+    retVal = self.loadChainFromString(pemData)
+    if not retVal['OK']:
       return retVal
-    return self.loadKeyFromString( pemData )
+    return self.loadKeyFromString(pemData)
 
-  def __getProxyExtensionList( self, diracGroup = False, rfc = False, rfcLimited = False ):
+  def __getProxyExtensionList(self, diracGroup=False, limited=False):
     """
     Get the list of extensions for a proxy
     """
     extList = []
-    extList.append( crypto.X509Extension( 'keyUsage',
-                                          'critical, digitalSignature, keyEncipherment, dataEncipherment' ) )
-    if diracGroup and isinstance( diracGroup, self.__validExtensionValueTypes ):
-      extList.append( crypto.X509Extension( 'diracGroup', diracGroup ) )
-    if rfc or rfcLimited:
-      blob = [ [ "1.3.6.1.5.5.7.21.1" ] ] if not rfcLimited else [ [ "1.3.6.1.4.1.3536.1.1.1.9" ] ]
-      asn1Obj = crypto.ASN1( blob )
+    extList.append(crypto.X509Extension('keyUsage',
+                                        'critical, digitalSignature, keyEncipherment, dataEncipherment'))
+    if diracGroup and isinstance(diracGroup, self.__validExtensionValueTypes):
+      extList.append(crypto.X509Extension('diracGroup', diracGroup))
+    if limited:
```

Comment:
What do `limited` means here? Is it different from `rfcLimited` that was before?

Reply:
No, the same thing, just the naming. rfcLimited was RFC and limited, to distinguish from non-RFC and limited. Now all are RFC, so that's simply limited.